docs/cli.rst

```diff
@@ -23,9 +23,9 @@ For convenience and added security you can set the username, password and DHuS U
   export DHUS_PASSWORD="<password>"
   export DHUS_URL="<api_url>"
 
-  sentinelsat -g <geojson>
+  sentinelsat -g <search_polygon.geojson>
 
-Search areas are provided as GeoJSON polygons, which can be created with
+Search areas (i.e. ``search_polygon.geojson`` ) are provided as GeoJSON polygons, which can be created with
```

Comment:
Strictly speaking, we also support `MultiPolygon` and `Point`, not just `Polygon`s. So instead of `GeoJSON polygons`, maybe write `GeoJSON files` here.